package.json

```diff
@@ -26,79 +26,82 @@
-    "ember-cli-fastboot": "2.1.1",
-    "ember-cli-htmlbars": "^4.3.1",
+    "ember-cli-fastboot": "^2.2.2",
+    "ember-cli-htmlbars": "^5.1.2",
     "ember-cli-inject-live-reload": "2.0.2",
-    "ember-cli-meta-tags": "5.3.0",
-    "ember-cli-moment-shim": "^3.7.1",
-    "ember-cli-qunit": "^4.0.0",
+    "ember-cli-meta-tags": "^6.0.0",
+    "ember-cli-moment-shim": "^3.8.0",
+    "ember-cli-qunit": "^4.4.0",
     "ember-cli-release": "^1.0.0-beta.2",
     "ember-cli-sass": "10.0.1",
     "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "2.0.2",
     "ember-cli-uglify": "3.0.0",
-    "ember-composable-helpers": "3.1.1",
-    "ember-concurrency": "1.1.6",
-    "ember-data": "~3.17.1",
-    "ember-data-storefront": "^0.17.0",
+    "ember-composable-helpers": "^4.1.3",
+    "ember-concurrency": "^1.1.7",
+    "ember-data": "^3.19.0",
+    "ember-data-storefront": "^0.17.2",
     "ember-decorators": "^6.1.1",
     "ember-export-application-global": "^2.0.1",
-    "ember-fetch": "7.1.0",
-    "ember-font-awesome": "4.0.0-alpha.4",
-    "ember-i18n": "^5.2.0",
+    "ember-fetch": "^8.0.1",
+    "ember-font-awesome": "^4.0.0-rc.4",
+    "ember-intl": "^5.2.0",
     "ember-lazy-image": "https://github.com/mcfiredrill/ember-lazy-image#update-dependencies",
     "ember-load-initializers": "^2.1.1",
     "ember-lodash": "4.19.5",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-moment": "^7.8.1",
-    "ember-page-title": "5.2.1",
+    "ember-moment": "^8.0.0",
+    "ember-page-title": "^5.2.3",
     "ember-parachute": "1.0.2",
-    "ember-power-select": "3.0.4",
+    "ember-power-select": "^4.0.2",
     "ember-qunit": "^4.6.0",
-    "ember-resolver": "^7.0.0",
+    "ember-resolver": "^8.0.0",
     "ember-rollbar-client": "^0.9.0",
     "ember-service-worker": "0.8.4",
     "ember-service-worker-asset-cache": "^0.6.4",
     "ember-service-worker-cache-fallback": "^0.6.2",
     "ember-service-worker-cache-first": "^0.6.3",
     "ember-service-worker-index": "^0.7.0",
     "ember-sinon-qunit": "^5.0.0",
-    "ember-source": "~3.15.0",
+    "ember-source": "^3.19.0",
     "ember-truth-helpers": "2.1.0",
     "ember-web-app": "^4.0.0",
     "ember-welcome-page": "^4.0.0",
     "emojione": "git://github.com/mcfiredrill/emojione#animated-emoji",
-    "eslint-plugin-ember": "7.10.1",
+    "eslint-plugin-ember": "^8.6.0",
     "eslint-plugin-node": "11.1.0",
     "fastboot-app-server": "2.0.0",
-    "jstimezonedetect": "^1.0.6",
+    "jstimezonedetect": "^1.0.7",
     "loader.js": "^4.2.3",
-    "phoenix": "1.4.14",
+    "phoenix": "^1.5.3",
     "qunit-dom": "1.2.0",
-    "sass": "^1.14.1",
+    "sass": "^1.26.8",
     "sinon": "^9.0.2",
     "stylelint-config-standard": "^20.0.0",
-    "textcomplete": "^0.17.1",
-    "video.js": "7.7.6"
+    "textcomplete": "^0.18.1",
+    "video.js": "^7.8.2"
+  },
+  "resolutions": {
+    "babel-plugin-ember-modules-api-polyfill": "2.13.0"
```

Comment:
This has to be here to force yarn to resolve `babel-plugin-ember-modules-api-polyfill`. Even though semantic versioning is supposed to ensure that patch changes don't break anything, they broke something. See [here](https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill/issues/112)